src/test/java/seedu/taassist/model/session/SessionTest.java

```diff
@@ -75,12 +75,10 @@ public void equals_diffValues_returnsFalse() {
 
     @Test
     public void isValidSessionName() {
-        // special characters and whitespaces are allowed
-        assertTrue(Session.isValidSessionName("+/~@#$%^&* "));
-
-        // sessions names beginning with whitespaces are not allowed
-        assertFalse(Session.isValidSessionName(" Assignment 1"));
-
+        // underscores and whitespace are allowed
+        assertTrue(Session.isValidSessionName("_LAB 1_"));
+        // special characters are not allowed
+        assertFalse(Session.isValidSessionName("+/~@#$%^&* "));
```

Comment:
Based on EP we shouldn't have so many invalid symbols in a single test case, but I guess the intention is just to show that the method will return false, so I guess this is fine.

Reply:
Yea,, we should look into EP in #241 but this should suffice for now